### PR TITLE
max relayer threshold by default

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: cb-sol-cli/cmd/deploy.js [hunk 1]
```diff
@@ -144,11 +144,13 @@ async function deployBridgeContract(args) {
     // Create an instance of a Contract Factory
     let factory = new ethers.ContractFactory(constants.ContractABIs.Bridge.abi, constants.ContractABIs.Bridge.bytecode, args.wallet);
 
+    const relayerThreshold = args.relayerThreshold === undefined ? "100000000000000000000" : args.relayerThreshold;
```
Comment (AndreiNavoichyk) [inline cb-sol-cli/cmd/deploy.js:147]: Double-check which type has a relayerThreshold in the SC.

Comment (karthikeyan-cere) [reply cb-sol-cli/cmd/deploy.js:147]: In smart contrat the relayer Threshold is uint256.  Which value should I need to pass here as default.

Comment (AndreiNavoichyk) [reply cb-sol-cli/cmd/deploy.js:147]: 1. Use default value in the line 10.
2. The max value it 2**256-1. To calculate it use BigNumber.
3. Move this constant to the constants file.

Comment (karthikeyan-cere) [reply cb-sol-cli/cmd/deploy.js]: Update the code with default value 2, from contants

File: cb-sol-cli/constants.js [hunk 2]
```diff
@@ -62,3 +62,5 @@ module.exports.ERC721_RESOURCEID = ethers.utils.hexZeroPad((this.ERC721_ADDRESS
 module.exports.GENERIC_RESOURCEID = ethers.utils.hexZeroPad((this.CENTRIFUGE_ASSET_STORE_ADDRESS + ethers.utils.hexlify(this.DEFAULT_SOURCE_ID).substr(2)), 32);
 
 module.exports.ERC20_PROPOSAL_HASH = "0x19b14d095647bb784f237072e14df1133fbd2008c5039c469321d77099a7b6da"
+
+module.exports.RELAYER_THRESHOLD = 2;
```
Comment (AndreiNavoichyk) [inline cb-sol-cli/constants.js:66]: Rename to MAX_UINT256

Comment (karthikeyan-cere) [reply cb-sol-cli/constants.js]: renamed

File: cb-sol-cli/cmd/deploy.js (hunk 1 above)
Comment (karthikeyan-cere) [reply cb-sol-cli/cmd/deploy.js]: Updated the code to use max_unit256

File: cb-sol-cli/cmd/deploy.js [hunk 3]
```diff
@@ -148,7 +148,7 @@ async function deployBridgeContract(args) {
     let contract = await factory.deploy(
         args.chainId,
         args.relayers,
-        args.relayerThreshold,
+        relayerThreshold,
```
Comment (AndreiNavoichyk) [inline cb-sol-cli/cmd/deploy.js:151]: Why did you remove `args`?

Comment (karthikeyan-cere) [reply cb-sol-cli/cmd/deploy.js]: reverted the changes/

**Review by AndreiNavoichyk — APPROVED** [review]
